app/src/test/java/com/volchok/rocketapp/feature/home/presentation/HomeViewModelTest.kt

```diff
@@ -0,0 +1,76 @@
+package com.volchok.rocketapp.feature.home.presentation
+
+import com.volchok.rocketapp.feature.home.domain.OpenRocketInfoUseCase
+import com.volchok.rocketapp.library.api.domain.ObserveRocketsUseCase
+import com.volchok.rocketapp.library.api.model.home.RocketItem
+import com.volchok.rocketapp.library.data.model.Data
+import io.kotest.matchers.shouldBe
+import io.mockk.*
+import kotlinx.coroutines.Dispatchers
+import kotlinx.coroutines.ExperimentalCoroutinesApi
+import kotlinx.coroutines.flow.flowOf
+import kotlinx.coroutines.test.*
+import org.junit.After
+import org.junit.Before
+import org.junit.Test
+
+internal class HomeViewModelTest {
```

Comment:
- chybi otestovat pripad kdy vrati observeRocketsUseCase error
- chybi otestovat defaultni stav s loadingem

Reply:
Opravil jsem to , ale nejsem si uplne jisty ze to ma byt tahle

Reply:
Opraveno